src/main/webapp/app/localci/build-agents/build-agent-details/build-agent-details/build-agent-details.component.ts

```diff
@@ -28,12 +29,15 @@ export class BuildAgentDetailsComponent implements OnInit, OnDestroy {
     faExclamationCircle = faExclamationCircle;
     faExclamationTriangle = faExclamationTriangle;
     faTimes = faTimes;
+    faPause = faPause;
```

Comment:
I would suggest we make all icons here readonly
```suggestion
    readonly faPause = faPause;
```